public/index.html

```diff
@@ -2,6 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <meta property="og:image" content="shareable_image.png">
```

Comment:
The image URL should have the `PUBLIC_URL` prefix.
`content="%PUBLIC_URL%/shareable_image.png"`